.tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml

```
@@ -226,7 +251,10 @@ spec:
                 value: "gpt-3.5-turbo-1106"
               - name: CP_OFFLINE_TOKEN
                 value: $(cat /var/run/insights-stage-upload-offline-token/token)
-            image: registry.redhat.io/openshift4/ose-cli:latest
+              - name: KUBECONFIG
+                value: "/credentials/$(tasks.ols-install.results.kubeconfig)"
+            image: $(steps.get-service-image.results.service-image)
```

```suggestion
            image: $(params.serviceImage)
```

And maybe this would work, not sure though